dashboard/modules/job/data_types.py

```diff
@@ -9,6 +9,9 @@
 
 
 class JobStatus(str, Enum):
+    def __str__(self):
```

Comment:
is this necessary? doesn't this happen automatically if you extend str

Reply:
By default it's `JobStatus.RUNNING`, not `RUNNING`